com.st.STM32CubeIDE.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: com.st.STM32CubeIDE
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: stm32cubeide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --allow=devel
+  - --filesystem=host
+  - --device=all
+  - --socket=session-bus
```

Comment:
any reason why it requires access to the whole session bus?

Reply:
Yes, `WebkitGDBus.java` needs it.

Reply:
It was fixed in Eclipse: https://bugs.eclipse.org/bugs/show_bug.cgi?id=540060

I don't expected anything to happen but would be nice to report it to the vendor.

Reply:
I... I don't understand which one to contact :dizzy_face: 
https://www.st.com/content/st_com/en/contact-us.html